launch.go

```diff
@@ -171,6 +195,24 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath string) error
 		return fmt.Errorf("fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	log.Printf("Fetching Parent Build %d for Meta", b.ParentBuildID)
```

Comment:
I think it will not have parentBuildId. 
Example here: 1272 triggers 1273
https://api.screwdriver.cd/v4/builds/1272 does not have parentBuildId
https://api.screwdriver.cd/v4/builds/1273 has parentBuildId as 1272